integ-test/src/test/resources/big5/queries/range_field_conjunction_big_range_big_term_query.ppl

```diff
@@ -1,5 +1,3 @@
 source = big5
-| where `process.name` = 'systemd'
-  and `metrics.size` >= 1
-  and `metrics.size` <= 1000
+| where `process.name` = 'systemd' and `metrics.size` >= 1 and `metrics.size` <= 100
```

Comment:
https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/big5/queries/range_field_conjunction_big_range_big_term_query.json

Reply:
Looks like there are difference sources of Big5 DSL. The original PPL was translated from the DSL listed above. But @noCharger seems find another source of Big5 DSL.

Reply:
Fix https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/2fbd2c7a96a3844bf41c12799e28bc539739d609/big5/operations/ppl.json#L178?